client/src/components/Dialogs/TimerDialog.vue

```diff
@@ -60,24 +44,33 @@
         outlined
         class="mr-2"
         @click="playPauseTimer"
-      >{{
-          pause ? "Play" : "Pause"
-        }}</v-btn>
-      <v-btn
-        color="secondary darken-1"
-        outlined
-        @click="stopTimer"
-      >Stop</v-btn>
+      >
+        <v-icon dark>
+          {{ pause ? "mdi-play" : "mdi-pause" }}
+        </v-icon>
+        {{ pause ? "Play" : "Pause" }}</v-btn
+      >
+      <v-btn color="secondary darken-1" outlined @click="stopTimer">
+        <v-icon dark> mdi-stop </v-icon>
+        Stop
```

Comment:
Stop button acts like a Reset button. I guess we need to add a Reset button as well in the future.

Reply:
Depends, on the countdown, it definitely doesn't act like a reset because if it was reset it would go again in the starting time you choose not on zero, on the timer, reset and stop would do the same thing.